src/mbgl/gl/types.hpp

```diff
@@ -96,5 +96,11 @@ enum class UniformDataType : uint32_t {
     SamplerCube = 0x8B60,
 };
 
+enum class BufferUsageType : uint32_t {
+    StreamDraw = 0x88E0,
+    StaticDraw = 0x88E4,
+    DynamicDraw = 0x88E8,
+};
```

Comment:
Thanks! Can we also add `static_assert` for those values similar to the other values we're checking in context.cpp at the top?